frontend/src/modules/automation/pages/automation-list-page.vue

```diff
@@ -1,154 +1,208 @@
+import AppAutomationForm from '@/modules/automation/components/automation-form.vue';
 import AppAutomationListTable from '@/modules/automation/components/list/automation-list-table.vue';
-import AppWebhookForm from '@/modules/automation/components/webhooks/webhook-form.vue';
-import AppWebhookExecutionList from '@/modules/automation/components/webhooks/webhook-execution-list.vue';
-import { getWorkflowMax, showWorkflowLimitDialog } from '../automation-limit';
+import config from '@/config';
+import { AuthToken } from '@/modules/auth/auth-token';
+import { mapGetters } from '@/shared/vuex/vuex.helpers';
+import AppAutomationExecutions from '@/modules/automation/components/automation-executions.vue';
 
-export default {
-  name: 'AppAutomationListPage',
-  components: {
-    AppWebhookExecutionList,
-    AppAutomationListTable,
-    AppWebhookForm,
-  },
-  data() {
-    return {
-      newAutomation: {
-        type: 'webhook',
-        settings: {},
-      },
-      isAutomationDrawerOpen: false,
-      isExecutionsDrawerOpen: false,
-      automation: null,
-    };
+const options = ref([
+  {
+    label: 'All',
+    value: 'all',
   },
-  computed: {
-    ...mapGetters({
-      loading: 'automation/loading',
-      filter: 'automation/filter',
-      count: 'automation/count',
-      currentTenant: 'auth/currentTenant',
-    }),
+  {
+    label: 'Slack notifications',
+    value: 'slack',
   },
-  async created() {
-    await this.doFetch({
-      filter: { ...this.filter },
-      rawFilter: { ...this.filter },
-    });
-  },
-  methods: {
-    ...mapActions({
-      doFetch: 'automation/doFetch',
-    }),
-    onOpenEditAutomationDrawer(automation) {
-      this.isAutomationDrawerOpen = true;
-      this.newAutomation = { ...automation };
-    },
-    onCloseAutomationDrawer() {
-      this.newAutomation = {
-        type: 'webhook',
-        settings: {},
-      };
-      this.isAutomationDrawerOpen = false;
-    },
-    onOpenExecutionsDrawer(automation) {
-      this.isExecutionsDrawerOpen = true;
-      this.automation = automation;
-    },
-    onCloseExecutionsDrawer() {
-      this.isExecutionsDrawerOpen = false;
-      this.automation = null;
-    },
-    async onAddWebhookClick() {
-      const isFeatureEnabled = FeatureFlag.isFlagEnabled(
-        FeatureFlag.flags.automations,
-      );
-
-      if (isFeatureEnabled) {
-        this.isAutomationDrawerOpen = true;
-      } else {
-        const planWorkflowCountMax = getWorkflowMax(
-          this.currentTenant.plan,
-        );
-
-        showWorkflowLimitDialog({ planWorkflowCountMax });
-      }
-    },
-    pluralize,
+  {
+    label: 'Webhooks',
+    value: 'webhooks',
   },
+]);
+const openAutomationForm = ref(false);
+const automationFormType = ref(null);
+const showAutomationExecutions = ref(null);
+const editAutomation = ref(null);
+
+const automationStore = useAutomationStore();
+const {
+  totalAutomations, filter, loadingAutomations, automations,
```

Comment:
Not sure why, but totalAutomations is not being reactive. I need to refresh the page to make it update. Could you take a look at this?

Reply:
Thanks for pointing,  will fix it.